ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/SandboxIndexAndWriteService.scala

```diff
@@ -117,19 +124,25 @@ private[sandbox] object SandboxIndexAndWriteService {
         templateStore,
         ledgerEntries,
       )
-    owner(MeteredLedger(ledger, metrics), participantId, timeProvider)
+    owner(
+      ledger = MeteredLedger(ledger, metrics),
+      participantId = participantId,
+      timeProvider = timeProvider,
+      enablePruning = false,
```

Comment:
Why can't we enable pruning for in-memory ledger?

Reply:
Maybe you can document it in the PR header why this one is left out

Reply:
It is a corer case that we should not spend any time debating at this point. It would have to go through another story if we wanted that support.

Reply:
fine by me. Just observed that there is no mention in the ticket or the PR regarding it.